cluster-provision

```diff
@@ -0,0 +1 @@
+vendor/kubevirtci/cluster-provision
```

Comment:
We don't appear to need the `cluster-provision` directory, it would be better not to include it in the first place

Reply:
Actually it would be better to include just the parts we want (pack8s and cluster-up) at the top-level, without any symlinks.  That avoids potential issues with go packaging tools that expect to manage the contents of the `vendor` directory.

Reply:
moved the kubevirt ci out of vendor, had to force push the whole sequence of commits. Please review again.